.github/scripts/create-repo.py

```diff
@@ -106,3 +107,10 @@
 
 with (REPO_DIR / "index.min.json").open("w", encoding="utf-8") as f:
     json.dump(index_min_data, f, ensure_ascii=False, separators=(",", ":"))
+
+with (REPO_DIR / "index.html").open("w", encoding="utf-8") as f:
+    f.write('<!DOCTYPE html>\n<html>\n<head>\n<meta charset="UTF-8">\n<title>apks</title>\n</head>\n<body>\n<pre>\n')
+    for entry in index_data:
+        apk_escaped = 'apk/' + html.escape(entry["apk"])
+        f.write(f'<a href="{apk_escaped}">{apk_escaped}</a>\n')
```

Comment:
nitpick: put the `apk/{}` url in href attribute but keep apk name as the link name